### PR TITLE
Add mujs

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

**Comment by jan-janssen** [comment]
@conda-forge/help-c-cpp

File: recipes/mujs/test.c [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (ocefpaf) [inline recipes/mujs/test.c:1]: Is this file supposed to be empty? If so, can you create it as part of the test?

Comment (jan-janssen) [reply recipes/mujs/test.c]: Thanks for the recommendation, I updated the test.

File: recipes/mujs/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+  host:
+    - readline
+
+test:
+  commands:
+    - touch test.c; mujs -s test.c
```
Comment (ocefpaf) [inline recipes/mujs/meta.yaml:27]: This should ensure that the second command only happens if the first one is successful. 
```suggestion
    - touch test.c && mujs -s test.c
```

**Pull request merged**